broker/l2tp_broker.py

```diff
@@ -393,6 +393,14 @@ def __init__(self, manager, port):
     self.next_session_id = 1
     self.id = -1
     self.keep_alive()
+    self.__keepalive_seqno = 0
+
+  @property
+  def _keepalive_seqno(self):
```

Comment:
Also it is great to have verbose names. This could be `nextSequenceNumber` and not reason for a property.